RepeatExplorer2 clustering is a computational pipeline for unsupervised identification of repeats from unassembled sequence reads. 
The pipeline uses low-pass whole genome sequence reads and performs graph-based clustering. 
Resulting clusters, representing all types of repeats, are then examined to identify and classify into repeats groups.

The analysis requires either single or paired-end reads generated by whole genome shotgun sequencing provided as a single fasta-formatted file.
Generally, paired-end reads provide significantly better results than single reads. 
Reads should be of uniform length (optimal size range is 100-200 nt) 
and the number of analyzed reads should represent less than 1x genome equivalent (genome coverage of 0.01 - 0.50 x is recommended). 
Reads should be quality-filtered (recommended filtering : quality score >=10 over 95% of bases and no Ns allowed)
and only complete read pairs should be submitted for analysis. 
