The following software is included in the Dfam TE Tools container (version 1.93):

RepeatModeler	2.0.7	https://github.com/Dfam-consortium/RepeatModeler
RepeatMasker	4.2.0	http://www.repeatmasker.org/RMDownload.html
coseg	0.2.3	http://www.repeatmasker.org/COSEGDownload.html
RMBlast	2.14.1	http://www.repeatmasker.org/RMBlast.html
HMMER	3.4	http://hmmer.org/
TRF	4.09.1	https://github.com/Benson-Genomics-Lab/TRF
RepeatScout	1.0.7	https://github.com/Dfam-consortium/RepeatScout
RECON	1.08	http://www.repeatmasker.org/RepeatModeler/RECON-1.08.tar.gz
cd-hit	4.8.1	https://github.com/weizhongli/cdhit
genometools	1.6.4	https://github.com/genometools/genometools
LTR_retriever	2.9.0	https://github.com/oushujun/LTR_retriever/
MAFFT	7.471	https://mafft.cbrc.jp/alignment/software/
NINJA	1.00-cluster_only	https://github.com/TravisWheelerLab/NINJA
UCSC utilities*	v413	http://hgdownload.soe.ucsc.edu/admin/exe/
RepeatAfterMe	0.0.7	https://github.com/Dfam-consortium/RepeatAfterMe
* Selected tools only: faToTwoBit, twoBitInfo, twoBitToFa

I will be using for comparison:
1) RepeatModeler
2) RepeatScout
3) RECON

How to do it:
1) RepeatModeler
singularity exec --bind ${PWD}:/data/ dfam-tetools-latest.sif BuildDatabase -name genome1 13.txt
singularity exec --bind ${PWD}:/data/ dfam-tetools-latest.sif RepeatModeler -database genome1 --threads 20

Finished running in 4 minutes (even though I ran it after I intiialized RepeatExplorer, and it finished earlier). Saved results into a folder with unique name so different runs wouldn't get intermixed. Accepst assembled reads, Doesn't require a specific extension.

3) RepeatExplorer
singularity exec --bind ${PWD}:/data/ repex_tarean.sif seqclust -v /data/re_output /data/m.fas
singularity exec --bind ${PWD}:/data/ repex_tarean.sif seqclust -v /data/re_output /data/sample.fas
Doesn't work! Need actual unassembled reads, preferrably reads 100-200 nt long

Trying on SRA file:
prefetch SRX29907886
fastq-dump --outdir ./ --gzip --skip-technical --readids --read-filter pass --dumpbase --split-3 SRR34774499.sra
seqtk seq -a SRR34774499_pass.fastq.gz > SRR34774499.fasta
singularity exec --bind ${PWD}:/data/ repex_tarean.sif seqclust -v /data/re_output /data/SRR34774499.fasta

RepearExplorer instruction states:
"The analysis requires either single or paired-end reads generated by whole genome shotgun sequencing provided as a single fasta-formatted file. Generally, paired-end reads provide significantly better results than single reads. Reads should be of uniform length (optimal size range is 100-200 nt) and the number of analyzed reads should represent less than 1x genome equivalent (genome coverage of 0.01 - 0.50 x is recommended). Reads should be quality-filtered (recommended filtering : quality score >=10 over 95% of bases and no Ns allowed) and only complete read pairs should be submitted for analysis. When paired reads are used, input data must be interlaced format as fasta file"

I am using single reads, 20000 reads ~250 nt long each which is suboptimal; nontheless, the program is running alhough it still didn't finish (the fasta file weighs 10 mbs)

Still didn't finish. Might come back on Monday and run it again or 

4) 
